coderd/audit_internal_test.go

```diff
@@ -0,0 +1,71 @@
+package coderd
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/coder/coder/v2/coderd/database"
+)
+
+func TestAuditLogDescription(t *testing.T) {
+	t.Parallel()
+	// nolint: paralleltest // no longer need to reinitialize loop vars in go 1.22
+	for _, tc := range []struct {
```

Comment:
could we move this out to a variable? generally not a huge fan of large inline variables like this